### PR TITLE
Edited curl command to handle quoting necessary to run on Windows 10.…

### LINKED ISSUE #3278: [BUG] mlflow tutorial: error running curl command on Windows 10
I am attempting to use the mlflow tutorial https://www.mlflow.org/docs/latest/tutorials-and-examples/tutorial.html#serving-the-model.

The curl invocation line when trying to call the served model is not working for me.

Windows 10.

Attempted on curl 7.55.1 and on 7.71.1.

Error:
```
curl -X POST -H "Content-Type:application/json; format=pandas-split" --data '{"columns":["alcohol", "chlorides", "citric acid", "density", "fixed acidity", "free sulfur dioxide", "pH", "residual sugar", "sulphates", "total sulfur dioxide", "volatile acidity"],"data":[[12.8, 0.029, 0.48, 0.98, 6.2, 29, 3.33, 1.2, 0.39, 75, 0.66]]}' http://127.0.0.1:1234/invocations
curl: (6) Could not resolve host: chlorides,
curl: (3) URL using bad/illegal format or missing URL
curl: (6) Could not resolve host: density,
curl: (3) URL using bad/illegal format or missing URL
curl: (3) URL using bad/illegal format or missing URL
curl: (6) Could not resolve host: pH,
curl: (3) URL using bad/illegal format or missing URL
curl: (6) Could not resolve host: sulphates,
curl: (3) URL using bad/illegal format or missing URL
curl: (3) unmatched close brace/bracket in URL position 17:
volatile acidity],data:[[12.8,
```

### Willingness to contribute
The MLflow Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the MLflow code base?

- [ ] Yes. I can contribute a fix for this bug independently.
- [x ] Yes. I would be willing to contribute a fix for this bug with guidance from the MLflow community.
- [ ] No. I cannot contribute a bug fix at this time.

### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **MLflow installed from (source or binary)**: binary
- **MLflow version (run ``mlflow --version``)**: 1.10.0
- **Python version**: 3.7.8
- **npm version, if running the dev UI**:
- **Exact command to reproduce**: curl -X POST -H "Content-Type:application/json; format=pandas-split" --data '{"columns":["alcohol", "chlorides", "citric acid", "density", "fixed acidity", "free sulfur dioxide", "pH", "residual sugar", "sulphates", "total sulfur dioxide", "volatile acidity"],"data":[[12.8, 0.029, 0.48, 0.98, 6.2, 29, 3.33, 1.2, 0.39, 75, 0.66]]}' http://127.0.0.1:1234/invocations

### Describe the problem
I am attempting to use the mlflow tutorial https://www.mlflow.org/docs/latest/tutorials-and-examples/tutorial.html#serving-the-model.

Simply following the tutorial and it does not work.

### Code to reproduce issue
the curl command above does not work

### Other info / logs
```
curl -X POST -H "Content-Type:application/json; format=pandas-split" --data '{"columns":["alcohol", "chlorides", "citric acid", "density", "fixed acidity", "free sulfur dioxide", "pH", "residual sugar", "sulphates", "total sulfur dioxide", "volatile acidity"],"data":[[12.8, 0.029, 0.48, 0.98, 6.2, 29, 3.33, 1.2, 0.39, 75, 0.66]]}' http://127.0.0.1:1234/invocations
curl: (6) Could not resolve host: chlorides,
curl: (3) URL using bad/illegal format or missing URL
curl: (6) Could not resolve host: density,
curl: (3) URL using bad/illegal format or missing URL
curl: (3) URL using bad/illegal format or missing URL
curl: (6) Could not resolve host: pH,
curl: (3) URL using bad/illegal format or missing URL
curl: (6) Could not resolve host: sulphates,
curl: (3) URL using bad/illegal format or missing URL
curl: (3) unmatched close brace/bracket in URL position 17:
volatile acidity],data:[[12.8,
```

### What component(s), interfaces, languages, and integrations does this bug affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [] `area/docs`: MLflow documentation pages
- [x] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/server-infra`: MLflow server, JavaScript dev server
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [x ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

### PR DESCRIPTION
**Pull request by extrospective**
Signed-off-by: Mark V. Andersen <markandersenphdgit@gmail.com>

## What changes are proposed in this pull request?

- Fixes  #3278
- Fix curl command to work for Windows

## How is this patch tested?

generated doc files successfully

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [x] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/server-infra`: MLflow server, JavaScript dev server
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: docs/source/tutorials-and-examples/tutorial.rst [hunk 1]
```diff
@@ -312,7 +312,7 @@ in MLflow saved the model as an artifact within the run.
 
       .. code-block:: bash
 
-          curl -X POST -H "Content-Type:application/json; format=pandas-split" --data '{"columns":["alcohol", "chlorides", "citric acid", "density", "fixed acidity", "free sulfur dioxide", "pH", "residual sugar", "sulphates", "total sulfur dioxide", "volatile acidity"],"data":[[12.8, 0.029, 0.48, 0.98, 6.2, 29, 3.33, 1.2, 0.39, 75, 0.66]]}' http://127.0.0.1:1234/invocations
+          curl -X POST -H "Content-Type:application/json; format=pandas-split" --data "{\"columns\":[\"alcohol\", \"chlorides\", \"citric acid\", \"density\", \"fixed acidity\", \"free sulfur dioxide\", \"pH\", \"residual sugar\", \"sulphates\", \"total sulfur dioxide\", \"volatile acidity\"],\"data\":[[12.8, 0.029, 0.48, 0.98, 6.2, 29, 3.33, 1.2, 0.39, 75, 0.66]]}" http://127.0.0.1:1234/invocations
```
Comment (harupy) [inline docs/source/tutorials-and-examples/tutorial.rst:315]: @extrospective Thanks for the contribution! Can you fix these lines as follows because showing only the escaped command would make it difficult to understand what the json data looks like?
 
```
# On Linux and macOS
curl -X ...

# On Windows
curl -X ...
```

File: docs/source/tutorials-and-examples/tutorial.rst [hunk 2]
```diff
@@ -382,7 +382,6 @@ in MLflow saved the model as an artifact within the run.
       .. code-block:: bash
 
           curl -X POST "http://127.0.0.1:8090/predict/" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"fixed acidity\": 6.2, \"volatile acidity\": 0.66, \"citric acid\": 0.48, \"residual sugar\": 1.2, \"chlorides\": 0.029, \"free sulfur dioxide\": 29, \"total sulfur dioxide\": 75, \"density\": 0.98, \"pH\": 3.33, \"sulphates\": 0.39, \"alcohol\": 12.8}"
-
```
Comment (harupy) [inline docs/source/tutorials-and-examples/tutorial.rst:385]: Can you revert this change because it's unrelated to this PR?

File: docs/source/tutorials-and-examples/tutorial.rst (hunk 1 above)
Comment (extrospective) [reply docs/source/tutorials-and-examples/tutorial.rst:315]: I thought about it, but decided since the purpose of the tutorial is not to introduce curl but to teach mlflow that it was not worth the added complexity.

Do you feel confident this is a better solution from the user's perspective?

If so I will implement as suggested.

File: docs/source/tutorials-and-examples/tutorial.rst (hunk 2 above)
Comment (extrospective) [reply docs/source/tutorials-and-examples/tutorial.rst]: Yes, oops.  Done below.

File: docs/source/tutorials-and-examples/tutorial.rst (hunk 1 above)
Comment (harupy) [reply docs/source/tutorials-and-examples/tutorial.rst:315]: @dmatrix What do you think?

Comment (dmatrix) [reply docs/source/tutorials-and-examples/tutorial.rst:315]: @extrospective thanks for fixing this. From the developer's perspective (and a reader of both UNIX and Windows), having two
separate callouts make it readable. It's not an uncommon practice in developer documentation to have separate callouts of CLI syntax between different operating systems. So I would lean toward:

```
# On Linux and macOS
curl -X ...

# On Windows
curl -X ...
```

Comment (extrospective) [reply docs/source/tutorials-and-examples/tutorial.rst:315]: I appreciate the thoughtful feedback.  

Since the change is compatible for Linux and macOS I had not come to the same conclusion.  It's just a less readable format with quoting.

But it might not be worth the time to opine further.  I will implement as proposed.  It will work for end users.
